api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -141,5 +142,19 @@ def update_app_list
         existing_apps.reload
       end
     end
+
+    def update_sub_tenants
+      return if params[:organization].key?(:sub_tenant_ids).blank?
+
+      sub_tenants = MnoEnterprise::SubTenant.where({'id.in' => params[:organization][:sub_tenant_ids]})
+
+      sub_tenants.to_a.each do |sub_tnt|
```

Comment:
This is not really scalable if we have a huge list for `client_ids`.
See MNOE-869 and the fix added in #532